bin/update-gilt-sbt-build.sh

```diff
@@ -10,5 +13,5 @@ while read -r LINE || [[ -n $LINE ]]; do
   git checkout -b "upgrade-to-g-s-b-$VERSION"
   sed -i "" -E 's/addSbtPlugin\("com.giltgroupe"[[:space:]]+%[[:space:]]+"gilt-sbt-build"[[:space:]]+%[[:space:]]+"[[:digit:]]+.[[:digit:]]+.[[:digit:]]+"\)/addSbtPlugin("com.giltgroupe" % "gilt-sbt-build" % "'"$VERSION"'")/g' project/build.sbt 
   git commit -a -m "NOJIRA-0: Upgrade gilt-sbt-build tp $VERSION"
-  git push origin HEAD:refs/for/master
+  git push origin HEAD:refs/for/$BRANCH
```

Comment:
Why would we want to push the changes for anything other than master (in regards to Gilt workflows since that's the only thing used)?